### DISCUSSION TITLE
filesystem.cpp(14): fatal error C1083 : Cannot open

### DISCUSSION THREAD

**Discussion by Fxztam** [neutralinojs/neutralinojs#759] [Q&A]
Hi,
I am trying to build it on Windows 10:
- installed Bost : Ok
- download latest : Ok
- ant : => filesystem.cpp(14): fatal error C1083 : _**Cannot open "atlstr.h": No such file or directory**_ 

~ no idea.

Thanks

**Comment by shalithasuranga** [comment]
Hey.. could you please check whether your Visual Studio installation has the Windows SDK?

**Comment by piggov** [comment]
@shalithasuranga Thank you, your suggestion worked for me!

A bit more details for others having this issue: (Win11 but should work on Win10 as well)

1. Open Visual Studio Installer
2. Find the version of visual studio you have installed (I have Community) and click Modify
3. In tab Workloads, section "Desktop & Mobile" select "Desktop development with C++"
4. I have enabled only these:
![image](https://user-images.githubusercontent.com/9749036/188437507-bd59eb46-00b0-4680-9dce-745f7023a5b2.png)
5. Click Modify

(or if you are just now installing Visual Studio, select the option above right from the installation)

@shalithasuranga Should we include this in the [contribution guide](https://neutralino.js.org/docs/contributing/framework-developer-guide/)?

**Comment by shalithasuranga** [comment]
Hey @Bonnev Thanks for posting detailed steps :) yeah sure, I think we can add it to the contribution guide page. If possible, please consider sending a sending pull-request since you've made the above list.

Thanks :tada:

**Reply by piggov** [reply]
@shalithasuranga I would love to! Here it is: [PR 240](https://github.com/neutralinojs/neutralinojs.github.io/pull/240)
